src/content/blog/2023/email-reader-view.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Email Reader View browser extension
+description: Display emails in a more readable format. Remove clutter and complicated layouts to help focus on reading the content of emails.
+author: mark-robbins
+pubDate: 2023-07-27
+published: true
+layout: "@layouts/BlogLayout"
+---
+
+We’re excited to launch our new Email Reader View browser extension. You can [install it from Chrome Web Store now](https://chrome.google.com/webstore/detail/email-reader-view/eekkbigfifdabokecancejangeoallck).
+
+You should also be able to use that link to install it on any Chromium based browsers such as Edge, Opera, Brave, etc.
+
+## What is reader view
+
+Email Reader View is a browser extension that allows the user to view an HTML email without all the clutter and distractions that can make reading difficult. It reduces the content to its most basic form of text but unlike a plain text email it keeps it’s semantic structure. Users can then apply their own styles to the text to suit their own preferences on things like font, text size, spacing and colors.
+
+This is inspired by web reader view features have become popular over the last few years, with built in versions in many browser as well as a number of browser extension. 
+
+## Why did we build it
+
+There are a number of goals we had with this project 
+
+### Help users with consuming content
+
+Accessibility is a big goal for us at the EMC. This tool could help a number of users consume email content more easily. If a user struggles to read a certain font, or if the text is too small to read, or if they experience glare or halo effects on text with certain color combinations these can all be fixed. We also have an option to replace images with their alt text so text in images can now be more readable (assuming the sender has added good quality alt text).
+
+Personally I use reader view on webpages a lot and since working on this project, I’ve started using it in email often too.
+
+### Help with email client rendering issues
+
+Sometimes emails can display badly due to various issue in different email clients or badly written code from the sender. Applying reader view removes all of the styling, simplifying the content and should mean the content is always readable despite these potential issues. A common issue is an email that isn’t built to be responsive can be scaled much better now using reader view. 
```

Comment:
There is an opportunity to mention newly introduced bugs by email clients that email developers haven't had the chance to address (like the recent background image bug on Gmail). White/light text on white/light background is an issue users recently encountered.

The browser extension provides a way to consume the content when rendering unexpectedly break.